thirdparty/DiskANN/include/diskann/utils.h

```diff
@@ -836,6 +836,10 @@ namespace diskann {
   // NOTE: Implementation in utils.cpp.
   void block_convert(std::ofstream& writr, std::ifstream& readr,
                      float* read_buf, _u64 npts, _u64 ndims);
+  
+  template<typename IN, typename OUT>
+  void convert_types_in_file(const std::string& inFileName,
```

Comment:
Where is the usage?

Reply:
Used in diskann，remove it first and add it next time.